gunicorn/config.py

```diff
@@ -1309,6 +1309,19 @@ class LogConfig(Setting):
     """
 
 
+class LogConfigDict(Setting):
+    name = "logconfig_dict"
+    section = "Logging"
+    validator = validate_dict
+    default = {}
+    desc = """\
+    The log config dictionary to use, using the standard Python logging
+    module's dictConfig format added in python 2.7.
```

Comment:
It might be better to remove "added in python 2.7" and print a warning if `dictConfig` is `None` in `gunicorn/glogging.py`.